core/tests/test-modules-mappings/acceptance/logged-out-user/click-all-buttons-on-about-page.txt

```diff
@@ -1,4 +1 @@
 core/templates/pages/about-page/about-page.module.ts
-core/templates/pages/android-page/android-page.module.ts
-core/templates/pages/classroom-page/classroom-page.module.ts
-core/templates/pages/library-page/library-page.module.ts
```

Comment:
@jnvtnguyen Is this change expected?

Reply:
@vojtechjelinek Yes they are as Akash stated above.

Reply:
Okay, makes sense.